frontend/src/features/carUploader/useCarForm.ts

```diff
@@ -0,0 +1,33 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import z from "zod";
```

Comment:
zod는 named import 방식이 안전하다고 합니다 !
```suggestion
import { z } from "zod";
```